src/main/java/com/gluonhq/substrate/SubstrateDispatcher.java

```diff
@@ -28,20 +28,19 @@
 package com.gluonhq.substrate;
 
 import com.gluonhq.substrate.model.ProcessPaths;
-import com.gluonhq.substrate.model.ProjectConfiguration;
+import com.gluonhq.substrate.model.PrivateProjectConfiguration;
 import com.gluonhq.substrate.model.Triplet;
 import com.gluonhq.substrate.target.*;
-import com.gluonhq.substrate.util.Logger;
 
 import java.io.File;
 import java.io.IOException;
 import java.nio.file.Path;
 import java.nio.file.Paths;
-import java.util.Arrays;
-import java.util.Objects;
-import java.util.Optional;
+import java.util.*;
```

Comment:
Make sure you set IntelliJ not to use wildcards in imports (set Preferences -> Editor > Code Style > Java > Imports tab set Class count to use import with '*' -> 99)

Reply:
I agree with avoiding wildcards (99 being ok, as in some cases there might be static imports of 100+ entities)

Reply:
Done